### PR TITLE
Feature/84 apply vocab parallel cross entropy

### LINKED ISSUE #84: Apply vocab parallel cross entropy for oslo models
## Describe a requested feature

- Apply vocab parallel cross entropy for oslo models.

### PR DESCRIPTION
**Pull request by loopinf**
## Title

- Add a few model and test on TP1D

## Description

- Add models to init and a test on TP1 vocab parallel crossentropy loss

## Linked Issues

- resolved #84

### CODE REVIEW

**Comment by hyunwoongko** [comment]
can you add all models not only gpt2 and bert?

**Comment by loopinf** [comment]
> 

This is the work already done and I'm almost done on every models

**Comment by hyunwoongko** [comment]
it's okay
